### PR TITLE
Feature/incident report/81290834

### PR DESCRIPTION
**Pull request by LeeroyDilim**
implemented barebones microservice of incident_reports. endpoint functionality is fully implemented. just needs mongo db implementation, staff front end, and permission validation.

### CODE REVIEW

File: src/incident_reports/requirements.txt
Comment (nateng98) [inline src/incident_reports/requirements.txt:1]: Have it install CORS as well, please add:
```
flask-cors==5.0.0
```

File: src/incident_reports/incident_reports_server/controllers/incident_reports_controller.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+import os
+
+from incident_reports_server.application.services import Services
+from incident_reports_server.factory.incident_report_factory import IncidentReportFactory
+from incident_reports_server.models.models import IncidentReportResponse, ResponseMessages
+from incident_reports_server.validators.incident_report_validator import IncidentReportValidator
+
+def create_app(persistence=None):
+    app = Flask(__name__)
+
```
Comment (nateng98) [inline src/incident_reports/incident_reports_server/controllers/incident_reports_controller.py:12]: Enable CORS so React and Angular can access the backend:
- Import cors
```py
from flask_cors import CORS
```
- Enable cors
```py
CORS(app)
```
_We let all origins can access incident_reports service for now_

**Review by nateng98 — DISMISSED** [review]
Look solid!
But please enable CORS so React and Angular can access incident reports service.

**Review by rainclouded — COMMENTED** [review]
Nice work. Just one comment

**Pull request merged**